### PR TITLE
Vet Clinic Database Exercise - Milestone 2: Query and Update animals table

### PR DESCRIPTION
**Pull request by AyoubM490**
In this milestone, I:
- Added a column "species" of type string to the Animals Table to the schema.sql file
- Inserted Animal data into the data.sql file
- Inside a transaction: 
  * Updated the animals' table by setting the species column to unspecified.
  * Verified that change was made. 
  * Then rolled back the change and verified that species columns went back to the state before the transaction.
- After the rollback verified if all records in the animals' table still exist.
- Inside a transaction: 
  * Deleted all animals born after Jan 1st, 2022.
  * Created a savepoint for the transaction.
  * Updated all animals' weight to be their weight multiplied by -1.
  * Rolled back to the savepoint
  * Updated all animals' weights that are negative to be their weight multiplied by -1.
  * Committed the transaction
- Wrote queries to answer the following questions:
  * How many animals are there?
  * How many animals have never tried to escape?
  * What is the average weight of animals?
  * Who escapes the most, neutered or not neutered animals?
  * What is the minimum and maximum weight of each type of animal?
  * What is the average number of escape attempts per animal type of those born between 1990 and 2000?
- Added the queries to queries.sql file

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -259,6 +259,60 @@ INSERT into animals (
...
+FROM animals
+WHERE date_of_birth > 'January 1, 2022';
+SAVEPOINT younger_deleted;
+UPDATE animals
+SET weight_kg = weight_kg * (-1);
+ROLLBACK TO younger_deleted;
+UPDATE animals
+SET weight_kg = weight_kg * (-1)
+WHERE weight_kg < 0;
+COMMIT;
```
Comment (mateo951) [inline data.sql]: Please be sure to add these transactions in your queries.sql file 👀

Everything that is a query that helps to analyze data should be saved in the queries.sql file.

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @AyoubMs, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: data.sql (hunk 1 above)
Comment (AyoubM490) [reply data.sql]: ok

**Review by Gambit142 — APPROVED** [review]
Hi @AyoubMs 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**